src/RSocketRequester.cpp

```diff
@@ -26,14 +26,6 @@ RSocketRequester::~RSocketRequester() {
   }
 }
 
-void RSocketRequester::closeSocket() {
```

Comment:
currently RSocketRequester::closeSocket is the only method to close one rsocket instance on the server (method used by RTGW today). If we remove it, what is the solution for allowing the server to close rsocket instance?